aea/agent.py

```diff
@@ -101,6 +107,7 @@ def __init__(
         self._timeout = timeout
 
         self._tick = 0
+        self._main_loop: BaseAgentLoop = None  # type: ignore
```

Comment:
It should be `Optional[BaseAgentLoop] = None` and then the `type ignore` should be removed

Reply:
done